lib/msal-browser/docs/performance.md

```diff
@@ -28,3 +28,25 @@ const msalInstance = new msal.PublicClientApplication({
     }
 });
 ```
+
+## Bypass authority metadata resolution
+
+By default, during the process of retrieving a token MSAL.js will make a network request to retrieve metadata from the authority configured for the request. If you would like to skip this network request, you can provide the required metadata in the configuration of `PublicClientApplication`.
+
+**Important:** It is your application's responsibility to ensure it is using correct, up-to-date authority metadata. Failure to do so may result in your application not working correctly.
```

Comment:
Should we explicitly mention that this is only if they choose to provide the values to `msal`?